CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Fixed
 * Subscribing to a message channel while unsubscribing is pending (#675)
 * Using ```Visible``` instead of ```Enabled``` to make sure RNSM continues updating when off (#702)
+* Fixed a bug caused by DontDestroyOnLoad of a networkbehaviour that carried a WinState netvar - now this state is passed by server to the PostGame scene and it then stores that state in the netvar, eliminating the need to preserve a NetworkBehaviour-bearing gameObject across scenes. (#724)
```

Comment:
Was this bug a thing in 1.3.0?

Reply:
not sure if it was released actually

Reply:
This wasn't an issue in 1.3.0, so this blurb can be removed since we've introduced it since, right?

Reply:
done! rephrased and moved to Changed